components/dropdown/dropdown-content-mixin.js

```diff
@@ -1007,11 +1103,11 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 			}
 		}
 		if (this.align === 'start' || this.align === 'end') {
-			const shift = Math.min((targetWidth / 2) - (20 + 16 / 2), 0); // 20 ~= 1rem, 16 = pointer size
+			const shift = Math.min((targetRect.width / 2) - (20 + 16 / 2), 0); // 20 ~= 1rem, 16 = pointer size
```

Comment:
super nit but since you added `pointerLength` that could be used for this `16` here (and the 20 could also be a variable as well since it's used here and now in `_getPointerPosition`)

Reply:
Sounds good. I will tidy that up with one of my next PRs.

Reply:
So I decided to at least update to use `pointerLength`. There appears to be other usage of `20` in the code, so I think I want to look at those as well - will do that separatly.